net/protocols.go

```diff
@@ -0,0 +1,8 @@
+package net
+
+import (
+	"github.com/libp2p/go-libp2p-protocol"
+)
+
+// FilecoinDHT is the protocol of the filecoin DHT.
+const FilecoinDHT protocol.ID = "/fil/kad/user-3"
```

Comment:
nit: `user` isn't my favorite name, especially since this protocol id will be deployed on nightly.  How about `devnet` or `dev`?

Reply:
` devnet` sounds good to me, addressed